cassandra/cqlengine/columns.py

```diff
@@ -168,6 +168,36 @@ def __init__(self,
         self.position = Column.instance_counter
         Column.instance_counter += 1
 
+    def __ne__(self, other):
+        if isinstance(other, Column):
+            return self.instance_counter != other.instance_counter
+        return NotImplemented
+
+    def __eq__(self, other):
+        if isinstance(other, Column):
+            return self.instance_counter == other.instance_counter
+        return NotImplemented
+
+    def __lt__(self, other):
+        if isinstance(other, Column):
+            return self.instance_counter < other.instance_counter
+        return NotImplemented
+
+    def __le__(self, other):
+        if isinstance(other, Column):
```

Comment:
Any thoughts about implementing these others in terms of ``__lt__`` and ``__eq__`` to avoid writing this type check so many times?